### ISSUE TITLE
Python detected in create-react-app, but no Python

### ISSUE THREAD

**Issue by arashd** [returntocorp/bento#106]
Make sure you are running the latest version of Bento.

**Describe the bug**
Bento says `create-react-app` has JS and Python, but there's no Python in the repo.

```bash
$ bento init
Detected project with Python and node-js

ls -R  | awk '
   /:$/&&f{s=$0;f=0}
   /:$/&&!f{sub(/:$/,"");s=$0;f=1;next}
   NF&&f{ print s"/"$0 }' | grep "\.py"

# no output

```

**Expected result**
That Bento would only report JS

**Additional context**
bento/0.1.1b4
running on `github.com/facebook/create-react-app@6533a6db`

**Comment by nbrahms** [comment]
Hi @arashd ...

I'm having trouble reproducing this. On create-react-app I get:
```
➜  create-react-app git:(master) ✗ bento init
Creating default configuration at .bento.yml
✔ Created .bento.yml. Please check this file in to source control.

Detected project with node-js

✔ Bento is initialized on your project.
➜  create-react-app git:(master) ✗```

Did you run on a clean version of the project?

**Comment by arashd** [comment]
Confirmed fixed in `bento/0.1.1b6`. Do we know how this could've happened. Both Luke and I reproduced this on `facebook/create-react-app@6533a6db4`
